Apps/Playground/CMakeLists.txt

```diff
@@ -134,6 +133,11 @@ if(WIN32)
         PRIVATE "shlwapi.lib")
 endif()
 
+# See https://gitlab.kitware.com/cmake/cmake/-/issues/23543
+# If we can set minimum required to 3.26+, then we can use the `copy -t` syntax instead.
+add_custom_command(TARGET Playground POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E $<IF:$<BOOL:$<TARGET_RUNTIME_DLLS:Playground>>,copy,true> $<TARGET_RUNTIME_DLLS:Playground> $<TARGET_FILE_DIR:Playground>)
```

Comment:
This code replaces the need for the CMake extension function `target_link_to_dependencies` and associated.

Reply:
call to `target_link_to_dependencies` removed.

Reply:
We should also remove the related functions in CMakeExtensions repo itself.